### PR TITLE
anyenv 1.0.1 (new formula)

### PR DESCRIPTION
**Pull request by riywo**
Add anyenv (retry of #36183 )

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Now I transferred notable repository and there is no error from audit.

### CODE REVIEW

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/anyenv.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  desc "All in one for **env"
+  homepage "https://anyenv.github.io/"
+  url "https://github.com/anyenv/anyenv/archive/v1.0.0.tar.gz"
+  sha256 "f922f3d6f319dcbf36a16f392c9d48c2f6338d0f0cc087ca965a8f1bc12a83a4"
+
+  def install
+    prefix.install %w[bin completions libexec]
+  end
+
+  def caveats
```
[the remark this answers is not in the document]
Comment (riywo) [reply Formula/anyenv.rb]: It is removed in the new commit.

File: Formula/anyenv.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      eval "$(anyenv init -)"
+
+      You can initialize install manifests directory by:
+
+      anyenv install --init [git-url] [git-ref]
+    EOS
+  end
+
+  test do
+    shell_output("eval \"$(#{bin}/anyenv init -)\" && anyenv --version")
```
[the remark this answers is not in the document]
Comment (riywo) [reply Formula/anyenv.rb]: I added more integrated tests in the latest commit.

**Comment by riywo** [comment]
Thank you for the review. I've addressed two comments in the latest commit.

**Comment by riywo** [comment]
@fxcoudert Why is this stuck pending?

**Comment by riywo** [comment]
@fxcoudert Is there any clue to address the failure? `brew test anyenv` is passing on my local laptop.

**Comment by SMillerDev** [comment]
The audit is failing, not the `brew test`

**Comment by riywo** [comment]
@SMillerDev Thanks, I fixed audit errors and now it has passed.

**Comment by SMillerDev** [comment]
@fxcoudert happy with these changes?

**Comment by riywo** [comment]
@fxcoudert Do I still need to change anything?

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @riywo for the pull request

**Pull request closed without merging**